hasher-matcher-actioner/terraform/api/main.tf

```diff
@@ -83,7 +84,7 @@ data "aws_iam_policy_document" "api_root" {
   }
   statement {
     effect    = "Allow"
-    actions   = ["dynamodb:GetItem", "dynamodb:Query", "dynamodb:Scan", "dynamodb:PutItem", "dynamodb:UpdateItem"]
+    actions   = ["dynamodb:GetItem", "dynamodb:Query", "dynamodb:Scan", "dynamodb:PutItem", "dynamodb:DeleteItem"]
```

Comment:
You need update AND delete, right?

Reply:
actually for now we just need put_item and delete_item, update_item is not needed for now